### PR TITLE
Build application manager image in CI

### PR DESCRIPTION
**Pull request by neosab**
We missed to build the app manager image and push to the local registry.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -67,6 +67,7 @@ run-dev: ## run the dev server
 
 .PHONY: linux
 linux: ## build the server binary
+	GOOS=linux go build -o bin/application-manager-linux ./cmd/application-manager
```
Comment (berndtj) [inline Makefile]: look at line 78

File: Makefile [hunk 2]
```diff
@@ -79,6 +80,7 @@ linux: ## build the server binary
 
 .PHONY: darwin
 darwin: ## build the server binary
+	GOOS=darwin go build -o bin/application-manager-darwin ./cmd/application-manager
```
Comment (berndtj) [inline Makefile]: line 91 is wrong, but also kind of a dup

File: Makefile [hunk 3]
```diff
@@ -100,6 +102,7 @@ images: linux ci-images
 
 .PHONY: ci-images
 ci-images:
+	scripts/images.sh application-manager $(BUILD)
```
Comment (berndtj) [inline Makefile]: line 113

File: Makefile [hunk 4]
```diff
@@ -112,6 +115,7 @@ ci-images:
 
 .PHONY: generate
 generate: ## run go generate
+	scripts/generate.sh application-manager ApplicationManager application-manager.yaml
```
Comment (berndtj) [inline Makefile]: line 125

**Comment by neosab** [comment]
Oh yeah my bad, I was on a stale branch and thought the app-manager wasn't added to Makefile. I think we now know what's happening so closing the PR.

**Pull request closed without merging**